test/eventlistener_test.go

```diff
@@ -180,17 +180,19 @@ func TestEventListenerCreate(t *testing.T) {
 	_, err = c.KubeClient.RbacV1().ClusterRoles().Create(
 		&rbacv1.ClusterRole{
 			ObjectMeta: metav1.ObjectMeta{Name: "my-role"},
-			Rules: []rbacv1.PolicyRule{
-				{
-					APIGroups: []string{"tekton.dev"},
-					Resources: []string{"clustertriggerbindings", "eventlisteners", "triggerbindings", "triggertemplates", "pipelineresources"},
-					Verbs:     []string{"create", "get"},
-				},
-				{
-					APIGroups: []string{""},
-					Resources: []string{"configmaps"},
-					Verbs:     []string{"get", "list", "watch"},
-				},
+			Rules: []rbacv1.PolicyRule{{
+				APIGroups: []string{triggersv1.GroupName},
+				Resources: []string{"clustertriggerbindings", "eventlisteners", "triggerbindings", "triggertemplates"},
+				Verbs:     []string{"create", "get"},
```

Comment:
I think this only needs `"get"`. I think that we can delete `"create"`, because the TriggerTemplate doesn't create any Triggers resources.